swarm/network/stream/common_test.go

```diff
@@ -62,6 +64,8 @@ var (
 	createStoreFunc   func(id discover.NodeID, addr *network.BzzAddr) (storage.ChunkStore, error)
 	getRetrieveFunc   = defaultRetrieveFunc
 	subscriptionCount = 0
+	globalStore       *db.GlobalStore
```

Comment:
To be able to potentially switch globalStore between mem and db implementations, this variable could be the interface:

```
var globalStore mock.GlobalStorer
```